### PR TITLE
tests: make it easer to override the location of the copy helper

### PR DESCRIPTION
**Pull request by nalind**
#### What type of PR is this?

/kind other

#### What this PR does / why we need it:

Make it possible to override the location of the `copy` test helper by setting its location in an environment variable, as we do for `imgtype`.

#### How to verify it

This is a tweak to how we cache images for integration tests.  They should continue to pass.

#### Which issue(s) this PR fixes:

Related to #3297.

#### Does this PR introduce a user-facing change?

```
None
```

### CODE REVIEW

**Comment by TomSweeneyRedHat** [comment]
LGTM
@edsantiago PTAL if you get a chance before your PTO

File: tests/helpers.bash [hunk 1]
```diff
@@ -2,6 +2,7 @@
 
 BUILDAH_BINARY=${BUILDAH_BINARY:-$(dirname ${BASH_SOURCE})/../bin/buildah}
 IMGTYPE_BINARY=${IMGTYPE_BINARY:-$(dirname ${BASH_SOURCE})/../bin/imgtype}
+COPY_BINARY=${COPY_BINARY:-$(dirname ${BASH_SOURCE})/../bin/copy}
```
Comment (edsantiago) [inline tests/helpers.bash:5]: I know this follows the `XXX_BINARY` convention, but at this point I'd like to suggest breaking away from that and using `BUILDAH_COPY_HELPER` instead. Something that makes it obvious and explicit that this is a custom buildah thing.

**Review by edsantiago — COMMENTED** [review]
One suggestion, and one request that I can't do inline: please remove the `COPY_OPTS=` definition in line 37.

File: tests/helpers.bash (hunk 1 above)
Comment (rhatdan) [reply tests/helpers.bash:5]: Could you open a separate PR to change IMGTYPE and COPY to the name you want.

Comment (edsantiago) [reply tests/helpers.bash:5]: >Could you open a separate PR to change IMGTYPE and COPY to the name you want.

It's too late for IMGTYPE: that change would need to be coordinated across gating tests on RHEL and Fedora, on all possible branches. That's just not sane.

I was hoping we could get this one done properly right now, so we could get things clean in gating tests.

**Pull request merged**